doc/7/controllers/auth/create-my-credentials/index.md

```diff
@@ -27,9 +27,10 @@ createMyCredentials(strategy, credentials, [options]);
 
 Additional query options
 
-| Property   | Type<br/>(default)              | Description                                                                  |
-| ---------- | ------------------------------- | ---------------------------------------------------------------------------- |
-| `queuable` | <pre>boolean</pre><br/>(`true`) | If true, queues the request during downtime, until connected to Kuzzle again |
+| Property   | Type<br/>(default)              | Description                                                                                                           |
+| ---------- | ------------------------------- | --------------------------------------------------------------------------------------------------------------------- |
+| `queuable` | <pre>boolean</pre><br/>(`true`) | If true, queues the request during downtime, until connected to Kuzzle again                                          |
+| `timeout`  | <pre>number</pre>               | Time (in ms) during which a request will still be waited to be resolved. Set it `-1` if you want to wait indefinitely |
```

Comment:
So at this action, this timeout has no default?

Reply:
There is a default timeout but IDK how to explain that the default timeout  is  the global timeout configured in the Kuzzle constructor if it has been configured or `-1` otherwise, meaning that there is no timeout